scene/gui/code_edit.cpp

```diff
@@ -3139,11 +3139,34 @@ void CodeEdit::_update_scroll_selected_line(float p_mouse_y) {
 	code_completion_force_item_center = -1;
 }
 
+/* Assumes both the new_options and the code_completion_options are sorted. */
+bool CodeEdit::_should_reset_selected_option_for_new_options(const Vector<ScriptLanguage::CodeCompletionOption> &new_options) {
```

Comment:
Method arguments must be prefixed with `p_` (same in the header).

```suggestion
bool CodeEdit::_should_reset_selected_option_for_new_options(const Vector<ScriptLanguage::CodeCompletionOption> &p_new_options) {
```